plugins/core/Standard/qRANSAC_SD/qRANSAC_SD_Commands.h

```diff
@@ -53,7 +53,7 @@ constexpr char PRIM_TORUS[] = "TORUS";
 
 struct CommandRANSAC : public ccCommandLineInterface::Command
 {
-	CommandRANSAC() : ccCommandLineInterface::Command("RANSAC", COMMAND_RANSAC) {}
+	CommandRANSAC() : ccCommandLineInterface::Command(QObject::tr("RANSAC"), COMMAND_RANSAC) {}
```

Comment:
I don't think any of these command "names" should be included in translation.

It looks like they are only used to display an internal that would only be useful for developers? Or am I missing something?

Reply:
They are displayed to console with the HELP command, some of the command names were already wrapped but it was a mixed bag.

```C++
else if (keyword == COMMAND_HELP)
		{
			print("Available commands:");
			for (auto it = m_commands.constBegin(); it != m_commands.constEnd(); ++it)
			{
				print(QString("-%1: %2").arg(it.key().toUpper(), it.value()->m_name));
			}
		}
```